en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authorization/opa-validation.md

```diff
@@ -0,0 +1,203 @@
+# Open Policy Agent (OPA) Validation
+
+The [Open Policy Agent (OPA)](https://openpolicyagent.org/) is an open source, general-purpose policy engine that unifies policy enforcement. In Choreo Connect, you can use OPA to enforce request to the API resources. Choreo Connect uses OPA’s policy evaluation REST API interface to communicate with OPA. Following diagram describes the request/response of OPA validation.
+
+!!! tip
+    You can deploy OPA server as a sidecar with Choreo Connect Runtime (Enforcer and Router) in a Kubernetes deployment, if you want to improve communication between Enforcer and OPA server.
+
+<img src="{{base_path}}/assets/img/deploy/mgw/choreo-connect-opa-overview.png" alt="Choreo Connect OPA request flow" width="700px"/>
```

Comment:
- Make the image zoomable when clicked
- Add the image size as a % instead of px